src/org/gk/slicing/SlicingEngine.java

```diff
@@ -1257,9 +1505,9 @@ public static void main(String[] args) {
             String dbHost = properties.getProperty("dbHost");
             if (dbHost == null || dbHost.trim().length() == 0)
                 dbHost = getInput("Please input the database host");
-            String dbName = properties.getProperty("dbName");
-            if (dbName == null || dbName.trim().length() == 0)
-                dbName = getInput("Please input the source database name");
+            String DbName = properties.getProperty("dbName");
```

Comment:
Why did you want to change the variable name from "dbName" to "DBName"? In Java, a variable should start with lower case.

Reply:
Lapse in judgement! Fixed in [5d0061c](https://github.com/reactome/CuratorTool/pull/20/commits/5d0061cf2e6db424efa382db7c7d1255250b379b).